webui/react/src/pages/TrialDetails/TrialInfoBox.tsx

```diff
@@ -22,10 +22,13 @@ interface Props {
   experiment: ExperimentDetails;
 }
 
-const hyperparamsView = (params: Record<string, React.ReactNode>) => {
+const hyperparamsView = (params: Record<string, HyperparameterValue>) => {
   return <List
     dataSource={Object.entries(params)}
-    renderItem={([ label, value ]) => <List.Item>{label}: {value}</List.Item>}
+    renderItem={([ label, value ]) => {
+      const textValue = isObject(value) ? JSON.stringify(value, null ,2) : value.toString();
```

Comment:
**nit:** minor lint problem, not sure if it will catch it. The comma in `null ,2` just needs to be `null, 2`

Reply:
it didn't catch it. I updated it.